config/initializers/inflections.rb

```diff
@@ -17,6 +17,7 @@
 ActiveSupport::Inflector.inflections(:en) do |inflect|
   inflect.irregular 'census', 'censuses'
   inflect.irregular 'supplemental_enrollment_data', 'supplemental_enrollment_data'
+  inflect.irregular 'hmis', 'hmis'
```

Comment:
It looks like this is causing some weird test failures

Reply:
or maybe its the extensions https://github.com/greenriver/hmis-warehouse/actions/runs/7389041046/job/20101087304